pkg/codegen/nodejs/gen.go

```diff
@@ -1012,7 +1029,7 @@ func (mod *modContext) getTypeImportsForResource(t schema.Type, recurse bool, ex
 		// If it's from another package, add an import for the external package.
 		if t.Resource != nil && t.Resource.Package != mod.pkg {
 			pkg := t.Resource.Package.Name
-			externalImports.Add(fmt.Sprintf("import * as %[1]s from \"@pulumi/%[1]s\";", pkg))
+			externalImports.Add(fmt.Sprintf("import * as %[1]s from \"@pulumi/%[2]s\";", fmt.Sprintf("pulumi%s", title(pkg)), pkg))
```

Comment:
Shouldn't this use `ProviderNameToModuleName` as well?

Reply:
Good question - that hasn't proven necessary in practice though it is possible there are cases where it matters. I'll take a second pass at this and update the PR tomorrow.

Reply:
Yup, you're right - addressed in 88c9fdf.